modules/ROOT/pages/editimage.adoc

```diff
@@ -0,0 +1,71 @@
+= Edit Image
+
+:title_nav: Edit Image
+:description: Image editing features for TinyMCE.
+:keywords: edit image rotate rotateleft rotateright flip flipv fliph editimage Edit Image options
```

Comment:
```suggestion
:keywords: edit image rotate rotateleft rotateright flip flipv fliph editimage imageoptions
```
`edit` and `image` are already in the keywords. Also `options` doesn't really make sense and the original `imageoptions` makes more sense to me since this relates to images so I've reverted that.